src/histolab/filters/morphological_filters.py

```diff
@@ -62,7 +66,23 @@ class RemoveSmallObjects(MorphologicalFilter):
     -------
     np.ndarray
         Mask with small objects filtered out
-    """
+
+
+
+    Example:
+        >>> from PIL import Image
+        >>> from histolab.filters.image_filters import RgbToGrayscale, OtsuThreshold
+        >>> from histolab.filters.morphological_filters import RemoveSmallObjects
+        >>> import os
+        >>> img_dir = os.path.join(os.path.abspath(os.getcwd()), "tests/fixtures/pil-images-rgb")
+        >>> image_rgb = Image.open(os.path.join(img_dir,"tcga-lung-rgb.png")) # Read an RGB image
+        >>> rgb_to_grayscale = RgbToGrayscale() # Instantiate the grayscale filter
```

Comment:
I would remove all these comments .... the idea was nice but the result is too cluttered